code/game/objects/structures/janicart.dm

```diff
@@ -16,7 +16,7 @@
 
 
 /obj/structure/janitorialcart/New()
-	create_reagents(100)
+	create_reagents(220)
```

Comment:
The container on the janitorial cart is a mop bucket, it should have exactly the same capacity as that

Actually making it a mop bucket and making it removable is something to consider for future, but for now at least, make the value the same